dotnet/Directory.Packages.props

```
@@ -68,13 +68,15 @@
     <PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="9.0.10" />
     <PackageVersion Include="Microsoft.Extensions.ServiceDiscovery" Version="$(AspireAppHostSdkVersion)" />
     <PackageVersion Include="Microsoft.Extensions.VectorData.Abstractions" Version="9.7.0" />
-    <!-- Vector Stores -->
+    <!-- Semantic Kernel -->
     <PackageVersion Include="Microsoft.SemanticKernel" Version="1.66.0" />
```

Good opportunity to bump all to 67.

```suggestion
    <PackageVersion Include="Microsoft.SemanticKernel" Version="1.67.0" />
```